tls_codec/Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2021"
 rust-version = "1.60"
 
 [dependencies]
-zeroize = { version = "1", default-features = false, features = ["alloc"] }
+zeroize = { version = "1", default-features = false, features = ["alloc", "zeroize_derive"] }
```

Comment:
Can you bump the version to `-pre.3`? Then we can release it right away.
I'd like to release this as `0.3` then next week.